aptos-move/framework/aptos-framework/sources/reconfiguration.spec.move

```diff
@@ -58,33 +72,49 @@ spec aptos_framework::reconfiguration {
         aborts_if !exists<Configuration>(@aptos_framework);
         let config_ref = global<Configuration>(@aptos_framework);
         aborts_if !(config_ref.epoch == 0 && config_ref.last_reconfiguration_time == 0);
+        ensures global<Configuration>(@aptos_framework).epoch == 1;
     }
 
     spec last_reconfiguration_time {
         aborts_if !exists<Configuration>(@aptos_framework);
+        ensures result == global<Configuration>(@aptos_framework).last_reconfiguration_time;
     }
 
     spec reconfigure {
-        use aptos_framework::aptos_coin;
+        use aptos_framework::coin::CoinInfo;
+        use aptos_framework::aptos_coin::AptosCoin;
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
 
         pragma verify_duration_estimate = 120; // TODO: set because of timeout (property proved)
 
         requires exists<stake::ValidatorFees>(@aptos_framework);
+        requires exists<CoinInfo<AptosCoin>>(@aptos_framework);
 
         include transaction_fee::RequiresCollectedFeesPerValueLeqBlockAptosSupply;
-        include features::spec_periodical_reward_rate_decrease_enabled() ==> staking_config::StakingRewardsConfigEnabledRequirement;
-        include features::spec_collect_and_distribute_gas_fees_enabled() ==> aptos_coin::ExistsAptosCoin;
-
+        include staking_config::StakingRewardsConfigRequirement;
         aborts_if false;
+
+        // The ensure conditions of the reconfigure function are not fully written, because there is a new cycle in it,
+        // but its existing ensure conditions satisfy hp.
         let success = !(chain_status::is_genesis() || timestamp::spec_now_microseconds() == 0 || !reconfiguration_enabled())
             && timestamp::spec_now_microseconds() != global<Configuration>(@aptos_framework).last_reconfiguration_time;
+        // property 3: Synchronization of NewEpochEvent counter with configuration epoch.
         ensures success ==> global<Configuration>(@aptos_framework).epoch == old(global<Configuration>(@aptos_framework).epoch) + 1;
+        ensures (success && event::counter<NewEpochEvent>(old(global<Configuration>(@aptos_framework)).events) <
```

Comment:
Please remove the property on event, this is proved because of inconsistency

Reply:
Could you double check whether it is actually removed?

Reply:
I removed the         ensures (success && event::counter<NewEpochEvent>(old(global<Configuration>(@aptos_framework)).events) <
            MAX_U64) ==>
            event::counter<NewEpochEvent>(global<Configuration>(@aptos_framework).events) ==
                event::counter<NewEpochEvent>(old(global<Configuration>(@aptos_framework)).events) + 1;

Reply:
Do you means this item should be keep as :

Reply:
No, current status looks good to me